.github/workflows/checks.yml

```diff
@@ -45,9 +45,30 @@ jobs:
     name: runner / uv-lock
     runs-on: ubuntu-latest
     steps:
-      - name: Check out repository
-        uses: actions/checkout@v4
-      - name: Install uv
-        uses: astral-sh/setup-uv@v6
+      - uses: actions/checkout@v4
+      - uses: astral-sh/setup-uv@v6
```

Comment:
why were the names removed from these jobs?

Reply:
@Kushagra651 : Can we keep the job names?